### PR TITLE
Initial work on Support page

### PR DESCRIPTION
**Pull request by retsimx**

### CODE REVIEW

File: docs/source/1-getting_started/Support.rst [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+Feel free to send a ticket through to our user support email address: hpc-support@swin.edu.au
+
+We do not write code for you, and we only provide best effort support.
+
+In your email to us please use the following template (Don't use this template for software installations).
+
+It may be beneficial to include the Job ID where applicable.
+
+```
+1. The issue you're having
```
Comment (jhurley) [inline docs/source/1-getting_started/Support.rst:19]: Add a line for JobID (if they have one)

File: docs/source/1-getting_started/Support.rst [hunk 2]
```diff
@@ -0,0 +1,27 @@
+User Support
+===
+
+Do you:-
+* have a problem that you're having trouble solving?
```
Comment (jhurley) [inline docs/source/1-getting_started/Support.rst]: related to OzSTAR

File: docs/source/1-getting_started/Support.rst [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+* have some software you need installed? 
+* feel unsure about the best or preferred way to launch a job or use OzSTAR? 
+* have some code that is crashing or not working as expected (But has worked for you before or on other systems)?
+* are experiencing something mysterious that you don't fully understand?
+
+Feel free to send a ticket through to our user support email address: hpc-support@swin.edu.au
+
+We do not write code for you, and we only provide best effort support.
+
+In your email to us please use the following template (Don't use this template for software installations).
```
Comment (jhurley) [inline docs/source/1-getting_started/Support.rst]: add a software request template

File: docs/source/1-getting_started/Support.rst [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+Do you:-
+* have a problem that you're having trouble solving? 
+* have some software you need installed? 
+* feel unsure about the best or preferred way to launch a job or use OzSTAR? 
+* have some code that is crashing or not working as expected (But has worked for you before or on other systems)?
+* are experiencing something mysterious that you don't fully understand?
+
+Feel free to send a ticket through to our user support email address: hpc-support@swin.edu.au
+
+We do not write code for you, and we only provide best effort support.
```
Comment (jhurley) [inline docs/source/1-getting_started/Support.rst]: Our goal is to provide best effort support and to get back to you as quickly as possible. However, some requests will be out of scope, e.g. don't ask us to rewrite your code for you, and will be politely declined.

**Review by jhurley — CHANGES_REQUESTED** [review]

File: docs/source/1-getting_started/Support.rst (hunk 1 above)
Comment (retsimx) [reply docs/source/1-getting_started/Support.rst:19]: Done.

**Review by jhurley — APPROVED** [review]

**Pull request merged**